### PR TITLE
TAG-4342

### PR DESCRIPTION
**Pull request by joaopgg**
Added
- tribe-button -> support for tabindex
- tribe-tab -> support for tabindex
- button example page

### CODE REVIEW

File: components/angular-tomitribe-fab/src/tomitribe-fab.ts [hunk 1]
```diff
@@ -96,17 +103,30 @@ module tomitribe_fab {
...
+                        if(isFocusOnTrigger() || hasNextFocusableElement()) {
+                            if (isFocusOnTrigger()) {
+                                //reset index
+                                scope.selectedIndex = 0;
+                            }
+                            moveDown($event);
+                        } else {
+                            //Going to exit
+                            if(scope.openOnFocus) {
+                                setMenuStatus(false, $event, true);
```
Comment (Dexmaster) [inline components/angular-tomitribe-fab/src/tomitribe-fab.ts:117]: I believe going to exit is not needed here as we are waiting for user to click esc or enter...
WDYT?

Comment (joaopgg) [reply components/angular-tomitribe-fab/src/tomitribe-fab.ts:117]: I tested this case, and we need to close the menu. 
Any suggestions to do it better are welcome.

**Review by Dexmaster — APPROVED** [review]
+1

File: components/angular-tomitribe-fab/src/tomitribe-fab.ts (hunk 1 above)
Comment (Dexmaster) [reply components/angular-tomitribe-fab/src/tomitribe-fab.ts:117]: Ok, was mistaken about one thing.

**Pull request merged**